src/Pages/CitiesPage/CitiesPage.js

```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { citiesData } from './citiesData';
-import CityItem from '../../Components/CityItem/CityItem.js';
+import CITIES from '../../MockData/cities.json';
```

Comment:
this should be small case
```suggestion
import cities from '../../MockData/cities.json';
```